modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -308,8 +390,21 @@ public ClientExtensionEntry updateClientExtensionEntry(
 			clientExtensionEntry.getTypeSettings(),
 			clientExtensionEntry.getType());
 
-		clientExtensionEntryLocalService.undeployClientExtensionEntry(
-			clientExtensionEntry);
+		undeployClientExtensionEntry(clientExtensionEntry);
```

Comment:
is this needed? later on we invoke the method in the entire cluster, so this is needed only if current node discarded in the multicast request

Just checked we use `skipLocal=true` in `ClusterRequest.createMulticastRequest()` method so this should be ok

Reply:
Yes, the other method only contacts slaves.